src/styles/page-card.scss

```diff
@@ -0,0 +1,93 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
+@import './stars';
+@import './colors';
+@import './card';
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+button,
+img {
+  padding: 0;
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.page {
+  margin: 0;
+  height: 100vh;
+
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__reviews {
+  margin-top: 16px;
+
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+
+  &-count {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $bold-text-color;
+  }
+}
+
+.card__price {
+  margin-top: 24px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+  align-items: center;
+
+  &-title {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $normal-text-color;
+  }
+
+  &-value {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $bold-text-color;
+  }
+}
+
+.card__button {
```

Comment:
Selector `.card__button` should be defined in `card.scss` since it's a BEM element of the `.card` block.